Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs

```diff
@@ -92,7 +92,8 @@ protected override async Task OnInitializedAsync()
 
             ConditionalCssClasses
                 .AddIf("mdc-fab--mini mdc-fab--touch", () => Type == MBFloatingActionButtonType.Mini)
-                .AddIf("mdc-fab--extended", () => Type == MBFloatingActionButtonType.Extended);
+                .AddIf("mdc-fab--extended", () => Type == MBFloatingActionButtonType.Extended)
+                .AddIf("mdc-fab--exited", () => Exited);
```

Comment:
Maybe a demo of this feature would be nice? One that sets `Exited` on click?

Reply:
We have this demo already, but the FABs all start as visible so the FOUC doesn't show up on the website

Reply:
I had a different demo in mind. I'll quickly add it.